src/WebJobs.Script/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -55,6 +55,13 @@ public static bool IsAuthDisabled(this HttpRequestMessage request)
             return request.GetRequestPropertyOrDefault<bool>(ScriptConstants.AzureFunctionsHttpRequestAuthorizationDisabledKey);
         }
 
+        public static bool IsAuthorized(this HttpRequestMessage request, AuthorizationLevel level)
```

Comment:
nit: Could we have a better name for this to indicate that it matches the specified authorization level? (e.g. `HasAuthorizationLevel`)